core/modules/book/book.install

```diff
@@ -128,7 +128,9 @@ function book_update_1001() {
  */
 function book_update_1002() {
   $config = config('book.settings');
-  $config->set('book_links', array('book_reorder'));
+  $book_links = $config->get('book_links');
+  $book_links['book_reorder'] = 'book_reorder';
```

Comment:
This will result in a structure different from what you have before. 

Before: 

```json
"book_links": ["book_add_child", "book_reorder"]
```

With this new code:
```json
"book_links": [
    0 : "book_add_child", 
    "book_reorder" : "book_reorder"
]
```

Will this lead to issues?

Reply:
Oops

Reply:
Instead try?

```php
$book_links[] = 'book_reorder';
```

Reply:
I guess I'll have to check what kind of array it is before adding to it. Annoying

Reply:
Fixed